### PR TITLE
bug: stop row click propagation for table action, checkboxes and  clickable cells

### PR DESCRIPTION
**Pull request by lab900-frederico-teixeira**
Stops click propagation for table actions, table checkboxes and clickable cells

### CODE REVIEW

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts [hunk 1]
```diff
@@ -178,6 +178,10 @@ export class TableExampleComponent {
       cellTooltip: { text: (data) => data.nameLong, onlyOnOverflow: true },
       cellMaxWidth: '300px', // overrides the maxColumnWidth on the table
       columnOrder: 1,
+      click: (data, cell, mouseEvent) => {
+        mouseEvent.stopPropagation();
+        console.log('click cell content > function params:', { data }, { cell }, { mouseEvent });
```
Comment (lab900-winand-vandenbergh) [inline src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:183]: Remove log

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts [hunk 2]
```diff
@@ -224,12 +228,12 @@ export class TableExampleComponent {
     });
   }
 
-  public rowClick(event, row, i): void {
-    console.log(event, row, i);
+  public rowClick(row, i, event): void {
+    console.log('click row > function params:', { row }, { i }, { event });
```
Comment (lab900-winand-vandenbergh) [inline src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:232]: Remove log

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts [hunk 3]
```diff
@@ -224,12 +228,12 @@ export class TableExampleComponent {
...
   }
 
-  public filtered(e): void {
-    console.log(e);
+  public filtered(tableCells: TableCell[]): void {
+    console.log({ tableCells });
```
Comment (lab900-winand-vandenbergh) [inline src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:236]: Remove log

**Review by lab900-winand-vandenbergh — CHANGES_REQUESTED** [review]
Remove logs and change target branch to develop

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts (hunk 1 above)
Comment (lab900-frederico-teixeira) [reply src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:183]: That's for the example and for debugging, I think its ok

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts (hunk 2 above)
Comment (lab900-frederico-teixeira) [reply src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:232]: same ☝️

File: src/app/modules/showcase-ui/examples/table-example/table-example.component.ts (hunk 3 above)
Comment (lab900-frederico-teixeira) [reply src/app/modules/showcase-ui/examples/table-example/table-example.component.ts:236]: same ☝️

**Review by lab900-winand-vandenbergh — APPROVED** [review]

**Pull request merged**